newsletter-march-2023.rst

```diff
@@ -31,7 +31,7 @@ News and updates
   If you find an unofficial and unmaintained project,
   :doc:`please see our policy on unofficial and unmaintained projects <readthedocs:unofficial-projects>`.
 - 🐞 Custom 404s with Sphinx DirHTML builder `are now supported <https://github.com/readthedocs/sphinx-notfound-page/issues/215>`__.
-- 🐞 :doc:`Pull Request <readthedocs:guides/pull-requests>` builds should point to previews, not build pages. The fix will be deployed on March 7.
+- 🐞 :doc:`Pull Request <readthedocs:guides/pull-requests>` builds should point to previews, not build pages. The fix will be deployed on March 14.
```

Comment:
Not sure why we're specifying dates here? We can just remove this..

Reply:
```suggestion
- 🐞 :doc:`Pull Request <readthedocs:guides/pull-requests>` builds should point to previews, not build pages.
```

Reply:
In that case, we should remove it entirely since it's not deployed? Otherwise, it looks like something that's fixed.

The idea of having the date was that readers could tell that it's gonna get fixed.

Reply:
IMHO, we should not put things that haven't happened yet in the newsletter 😄

Reply:
Yea, that seems like the correct solution.

Reply:
Yup, I know, that's what I wrote on Slack :)

I'll get it done the right way :+1:

Reply:
Moved the whole thing to "Upcoming" and linked to the PR

Reply:
I don't think mentioning a "future bugfix" is useful, anyways. The upcoming section is useful to communicate, in a more generic way, what are our future plans, what we plan to work on, and what our users can expect from our platform.

In any case, I'm don't want to block this PR on this, but just to keep it in mind for future newsletters.